src/content/docs/dropins/all/layouts.mdx

```diff
@@ -0,0 +1,111 @@
+---
+title: Commerce block layouts
+description: Learn about how to configure commerce block layouts.
+---
+
+import { Tabs, TabItem } from '@astrojs/starlight/components';
+import Diagram from '@components/Diagram.astro';
+import Vocabulary from '@components/Vocabulary.astro';
+import Aside from '@components/Aside.astro';
+import Callouts from '@components/Callouts.astro';
+import Tasks from '@components/Tasks.astro';
+import Task from '@components/Task.astro';
+
+A drop-in component's layout is defined by an HTML fragment that controls where the drop-in's containers appear on the page. You can customize the layout as you would with any HTML, by using CSS and adding, removing, or rearranging the elements in the HTML. In this topic, we'll customize the product-details layout by adding the Product Recommendations block.
+
+## Big Picture
+
+This screenshot shows the product details page with the Product Recommendations block below the product gallery container.
+
+<Diagram caption="Add Product Recommendations block to the page">
+![Add Product Recommendations block to the page](@images/ProductDetailsLayout.png)
+</Diagram>
+
+## Customize commerce block layouts
+
+For this use case, we'll customize the product-details layout by adding the Product Recommendations block inside the product-details block, instead of below it.
```

Comment:
```suggestion
For this use case, we'll customize the product details layout by adding the Product Recommendations block inside the product details block, instead of below it.
```